wiki/Tournaments/GTS/AGTS_2019/en.md

```diff
@@ -0,0 +1,297 @@
+---
+tags:
+- AGTS 2019
+- AGTS2019
+---
+# Advanced Global Taiko Showdown 2019
+
+![AGTS 2019 Logo](img/logo.png)
+
+The **Advanced Global Taiko Showdown 2019** (***AGTS 2019***) is an advanced worldwide country-based double-elimination two-on-two osu!taiko tournament hosted by ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263) and ![][flag_FR] [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133). It is the first installment of the Advanced Global Taiko Showdown and part of the Global Taiko Showdown series.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-07-07/2019-07-21 |
+| Live Drawings | 2019-08-04 (15:00 UTC) |
+| Qualifiers Stage | 2019-08-10/2019-08-11 |
+| Group Stage | 2019-08-17/2019-08-18 |
+| Quarterfinals | 2019-08-24/2019-08-25 |
+| Semifinals | 2019-08-31/2019-09-01 |
+| Finals | 2019-09-07/2019-09-08 |
+| Grand Finals | 2019-09-14/2019-09-15 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Unique profile badge, exclusive team banner |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | exclusive team banner |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | exclusive team banner |
+
+## Organisation
+
+The Advanced Global Taiko Showdown 2019 is run by various community members by distributing the multitude of tasks into various fields of responsibility.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Management | ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263), ![][flag_FR] [TLQ\_Yoshii](https://osu.ppy.sh/users/7157133) |
+| Map Selectors | ![][flag_DE] [Nepuri](https://osu.ppy.sh/users/6637817), ![][flag_US] [Nifty](https://osu.ppy.sh/users/4956097) |
```

Comment:
these roles should be written singular (so "Map Selector", "Referee", etc)

Reply:
New ASC?

Reply:
it's just grammar. you have the singular "Position" as the header, and "Map Selectors" for example isn't a single position/role

your options here would be either "Map Selector" or "Map Selection". I think the first is more like a "position" and the second is more like a "role"... subtle distinction. if u go with the first then ig u should change "Management" to "Manager" as well